cluster-up/cluster/kind-k8s-sriov-1.17.0/manifests/network_config_policy.yaml

```diff
@@ -8,9 +8,8 @@ spec:
   mtu: 1500
   nodeSelector:
     sriov: "true"
-  numVfs: $NODE_PF_NUM_VFS
+  numVfs: 
```

Comment:
This is not a valid yaml

Reply:
This yaml file propose is a template, it is copied and patched later on with 
`numVfs` and `pfNames`